app/shop.py

```diff
@@ -0,0 +1,50 @@
+class Shop:
+    def __init__(self, name: str, price_list: dict, location: list) -> None:
+        self.name = name
+        self.price_list = price_list
+        self.location = location
+
+    def calculate_purchase_cost(self, products_to_buy: dict) -> float:
+        cost = 0
+
+        for key in products_to_buy.keys():
+            cost += round(self.price_list[key] * products_to_buy[key], 2)
+
+        return cost
+
+    @staticmethod
+    def print_recipe(recipe_data: list[tuple], customer_name: str,
+                     total_cost: float) -> None:
+        # current_time = datetime.now()
+        # date_string = f"Date: {current_time.strftime("%m/%d/%Y %H:%M:%S")}"
```

Comment:
remove comments

Reply:
Done. I've left them, because it looked strange, that I should use static date instead of current date.